edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Kubernetes/KubernetesEventIds.cs

```diff
@@ -11,6 +11,7 @@ public struct KubernetesEventIds
         public const int KubernetesExperimentalCreateOptions = EventIdStart + 600;
         public const int KubernetesModelValidation = EventIdStart + 700;
         public const int KubernetesServiceBuilder = EventIdStart + 800;
+        public const int SecretBackup = EventIdStart + 900;
```

Comment:
Not related to this pr, but why isn't this an enum?

Reply:
I don't know exactly.  I looked up some possibilities, and I didn't find a reason.